start.sh

```diff
@@ -25,5 +25,5 @@ else
     source "${APP_DIR}/global_env"
     source "${APP_DIR}/app_env"
 
-    exec node ${APP_DIR}/bin/www.js -- $PORT
+    NODE_ENV=production node_16 ${APP_DIR}/bin/www.js -- $PORT
```

Comment:
According to @sroberts2 , this should read:
`    NODE_ENV=production node_16 ${APP_DIR}/dist/bin/www.js -- $PORT`

See:
https://github.com/companieshouse/admin.orders.web.ch.gov.uk/blob/main/start.sh#L33

Reply:
Hi Tom, the build target is different in the two project as you can see [here](https://github.com/companieshouse/admin.orders.web.ch.gov.uk/blob/6869f7a20a3ad67ff9e8681c00bb93463a72bba6/Makefile#L55) and [here](https://github.com/companieshouse/overseas-entities-web/blob/94f7286e6a29d4502922ee7d1061a92570c0c6cc/Makefile#L38). In one we copy the `dist` directory in other the entire content.

Reply:
Out of curiosity what is the difference between `node_16` and `node` command?

Reply:
I asked Mark Ransome about this as I was curious too. He said: 
**_Yeah definitely stick with the node_16 wrapper in start.sh—when we deploy the application to the Mesos cluster there are no containers involved and the start script is the canonical way the service will be started so it needs to invoke the wrapper at that point to ensure we don’t use the default runtime (10)_**

Reply:
I defer to Mou's knowledge of this app, so will resolve this.